tests/net/ethernet_mgmt/src/main.c

```diff
@@ -109,8 +111,9 @@ static int eth_fake_set_config(struct device *dev,
 		break;
 	case ETHERNET_CONFIG_TYPE_QAV_DELTA_BANDWIDTH:
 	case ETHERNET_CONFIG_TYPE_QAV_IDLE_SLOPE:
-		/* Assumes just one priority queue - validate the id*/
-		if (config->qav_queue_param.queue_id != 0) {
+		queue_id = config->qav_queue_param.queue_id;
+
```

Comment:
you can get rid of that empty line as your if is testing the previous assignment

Reply:
That is right, thanks!